pages/_error.js

```diff
@@ -13,7 +13,7 @@ class Error extends React.Component {
     return (
       <>
         <PageHead />
-
```

Comment:
FYI this isn't reflected yet in deployments due to `https://github.com/zeit/now-builders/issues/234`